server/model-config.json

```diff
@@ -11,19 +11,19 @@
   },
   "Definition": {
     "public": false,
-    "dataSource": false
+    "dataSource": "db"
```

Comment:
We are intentionally setting `"dataSource": false`, these models are not persisted by any datasource. Some of them have their own persistence API (e.g. ConfigFile and Workspace), others are "abstract" base classes (models).

Please revert changes in this file too.